microcosm_pubsub/daemon.py

```diff
@@ -12,9 +12,10 @@
 
 class ConsumerDaemon(Daemon):
 
-    def __init__(self):
+    def __init__(self, event={}):
```

Comment:
Using a mutable default value for an argument is an antipattern (see [explanation](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments)). The simplest is to use `None` and update line 18 to 
```self.sqs_event = event or {}```

Reply:
Thanks!